tests/devices/test_default_mixed.py

```diff
@@ -301,6 +318,10 @@ def test_diagonal_kraus(self, ops, tol):
                 np.sqrt(p_1) * np.array([[0, 0], [0, 1]]),
             ],
         ),
+        (
+            PauliError("X", p, wires=0),
+            [np.sqrt(1 - p) * np.eye(2), np.sqrt(p) * np.array([[0, 1], [1, 0]])],
```

Comment:
Not choosing `p=0.5` should help make the test more valuable.
```suggestion
            PauliError("X", p0, wires=0),
            [np.sqrt(1 - p0) * np.eye(2), np.sqrt(p0) * np.array([[0, 1], [1, 0]])],
```